.github/workflows/format_linter_enforcer.yml

```diff
@@ -21,13 +21,12 @@ jobs:
       - name: Install YAPF, autope8 and flake8
         run: pip install yapf autopep8 flake8
 
-      - name: Run Flake
+      - name: Run Flake8
         run: flake8
 
       - name: Check formatting
         run: |
           autopep8 -ir .
           yapf -ir .
```

Comment:
It might be good to simply run precommit here, rather than running the precommit packages? See what we have in gpytorch: https://github.com/cornellius-gp/gpytorch/blob/981edd83a671e8dca9d67c91b188702354884a34/.github/workflows/run_test_suite.yml#L32